modules/miner/mine.go

```diff
@@ -188,13 +190,13 @@ func (m *Miner) FindBlock() (types.Block, bool, error) {
 // and is very slow.
 func (m *Miner) SolveBlock(blockForWork types.Block, target types.Target) (b types.Block, solved bool) {
 	b = blockForWork
-	for b.Nonce = 0; b.Nonce < iterationsPerAttempt; b.Nonce++ {
+	for i := 0; i < 255; i++ {
+		b.Nonce[0] = byte(i)
 		if b.CheckTarget(target) {
-			solved = true
-			return
+			return b, true
```

Comment:
it's only purpose is for testing. Maybe we should keep it a little more useful than it currently is though.